scripts/npm.sh

```diff
@@ -8,7 +9,7 @@ else
 	ROOT=$(dirname "$(dirname "$(readlink -f $0)")")
 
 	# if [ -z $npm_config_arch ]; then
-	# 	npm_config_arch=$(node -p process.arch)
+	# 	npm_config_arch=$(npm -p process.arch)
```

Comment:
This should be `node` not `npm`. Looks like it might have been mistakenly changed, can you change it back?